crates/rspack_plugin_javascript/src/visitors/dependency/scanner.rs

```diff
@@ -63,14 +63,25 @@ impl DependencyScanner<'_> {
             }
             if let Some(expr) = call_expr.args.get(0) {
               if expr.spread.is_none() {
+                // TemplateLiteral String
+                if let Expr::Tpl(tpl) = expr.expr.as_ref()  && tpl.exprs.is_empty(){
+                  let s = tpl.quasis.first().expect("should have one quasis").raw.to_string();
```

Comment:
`to_string` is not necessary

Reply:
atom seems not support convert to JSWord directly how do i convert

Reply:
`Atom` is `JsWord`. They are the same type.

Reply:
let me see.

Reply:
fixed